GPU/Vulkan/GPU_Vulkan.cpp

```diff
@@ -268,6 +268,11 @@ u32 GPU_Vulkan::CheckGPUFeatures() const {
 		features |= GPU_ROUND_DEPTH_TO_16BIT;
 	}
 
+	if (true) {
```

Comment:
Should probably use `g_Config.bStereoRendering` or something.  Just noting so it's not forgotten, realize this is debugging.

-[Unknown]